README.md

```diff
@@ -14,7 +14,12 @@ Dependencies
 -   [**mraa**](https://github.com/intel-iot-devkit/mraa): This is not on PyPI and require additonal install steps. Follow the install instruction on [sparkfun](https://learn.sparkfun.com/tutorials/installing-libmraa-on-ubilinux-for-edison) with a couple updates:
    -   The step `apt-get install python-dev` needs to be updated to `apt-get install python3-dev`
    -   The step `cmake .. -DBUILDSWIGNODE=OFF` needs to be updated to `cmake .. -DBUILDSWIGNODE=OFF -DBUILDPYTHON3=ON`
-   -   If `export PYTHONPATH=$PYTHONPATH:$(dirname $(find /usr/local -name mraa.py))` doesn't work then `export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.4/site-packages` should suffice`
+   -   When appending the python path in .bashrc as outlined in the instructions, there is a syntax error: `export PYTHONPATH=$PYTHONPATH:$(dirname $(find /usr/local -name mraa.py))` should have double quotes inside it, like so: `export PYTHONPATH=$PYTHONPATH:$(dirname $"(find /usr/local -name mraa.py)")` If that doesn't work, then `export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.4/site-packages` should suffice.
+   -   If using **rc.local** to start n.io, GPIO will not be accessible. This is due to the fact that the above step has you append the python path in **.bashrc**, which is loaded after **rc.local**. Since this is the case, n.io has already initialized before it knows where the mraa library is. There are two potential fixes for this:
+   -   *Either*:
+          -   Append the python path with the path to the `mraa.py` file & `_mraa.so` shared library in **rc.local**, rather than in **.bashrc**.   *--or--*
```

Comment:
The official library docs recommend making a sym link instead of actually moving them. https://github.com/intel-iot-devkit/mraa/blob/2e5e3eaea2750d231cbebf04549162fcb7a4b48b/docs/building.md#basic-build-steps

`sudo ln -s /usr/local/lib/python3.4/site-packages/* /usr/local/lib/python3.4/dist-packages`